parlai/__init__.py

```diff
@@ -4,4 +4,4 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-__version__ = '0.9.4'
+__version__ = '0.10.0'
```

Comment:
Ha I suppose you could import VERSION from setup.py?

Reply:
Nope, not distributed together lol